pgscatalog.match/src/pgscatalog/match/lib/matchresult.py

```diff
@@ -318,19 +338,16 @@ def full_variant_log(self, score_df, **kwargs):
         Multiple match candidates may exist for each variant in the original file.
         Describe each variant (one variant per row) with match metadata
         """
-        if not self._labelled:
-            self.df = self.label(**kwargs)
-            self._labelled = True
+        if self.match_candidates is None:
+            self._match_candidates = self.label(**kwargs)
 
-        if not self._filtered:
-            self.df, self.filter_summary = filter_scores(
-                scorefile=score_df,
-                matches=self.df,
-                min_overlap=kwargs.get("min_overlap", 0.75),
-                dataset=self.dataset,
+        if self.filtered_matches is None:
+            self._filter_summary, self._filtered_matches = self.filter(
+                score_df=score_df, min_overlap=kwargs.get("min_overlap", 0.75)
             )
-            self._filtered = True
 
         return make_logs(
-            scorefile=score_df, dataset=self.dataset, match_candidates=self.df
+            scorefile=score_df,
+            dataset=self.dataset,
+            match_candidates=self.match_candidates,
```

Comment:
This caused the log to drop variant information. Before we were making the log using filtered variants, now we're correctly using the labelled match candidate